zeebe-cluster/values.yaml

```diff
@@ -15,6 +15,7 @@ cpuThreadCount: "2"
 ioThreadCount: "2"
 pvcSize: "10Gi"
 pvcAccessModes: [ "ReadWriteOnce" ]
+logLevel: warn
```

Comment:
why warn?

Reply:
I imagine if you're running this in production, you don't care about anything but warning/errors. Possibly you'd want INFO, depending on your level of paranoia. Is debug or info level logging interesting for people running Zeebe who are not Zeebe devs/contributors/testers?

Reply:
I set it down to INFO; debug should be mostly for us, don't see it as useful in production.